automation/terraform/modules/kubernetes/testnet/helm.tf

```diff
@@ -13,6 +13,14 @@ data "local_file" "genesis_ledger" {
   ]
 }
 
+data "local_file" "libp2p_peers" {
+  for_each = toset([ for config in var.block_producer_configs : config.name ])
+  filename = "../../../keys/libp2p/${var.testnet_name}/${each.key}"
```

Comment:
Probably worth prepending the filename path with one of Terraform's path vars (https://www.terraform.io/docs/configuration-0-11/interpolation.html#path-information) to be more explicit about where the keys should be stored. Also, there's an `artifact_path` VAR that was added that you may be able to use.